assignment/week3/lunchMenu/src/pages/RecommendStartPage.jsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import RandomPage from './RandomPage';
+import PreferencePage from './PreferencePage';
+import { useState } from 'react';
+const RecommendStartPage = (props) => {
+  const [startRecommend, setStartRecommend] = useState(false);
+  const clickStart = () => {
+    setStartRecommend(true);
+  };
+  return (
+    <>
+      {startRecommend ? (
+        <>
+          {props.category === '취향대로 추천' ? <PreferencePage category={props.category} /> : null}
+          {props.category === '랜덤으로 추천' ? <RandomPage category={props.category} /> : null}
```

Comment:
조건에 부합할 때만 렌더링 하고 싶다면, 굳이 삼항연산자를 안쓰고 && 연산자를 이용해도 된답니다!
```suggestion
          {props.category === '취향대로 추천' && <PreferencePage category={props.category} />}
          {props.category === '랜덤으로 추천' && <RandomPage category={props.category} />}
```

이렇게하면 좌측의 조건이 맞을 때 우측의 컴포넌트를 렌더링하겠다! 가 됩니다

아니면 이렇게 해도 되겠는걸요..?
```suggestion
          {props.category === '취향대로 추천' ? <PreferencePage category={props.category} /> : <RandomPage category={props.category} />}
```

요거는 근데 만약에 세번째 카테고리가 생기는 상황을 가정한다면 유지보수에 좋지 않는 구조가 될것이에요-!

Reply:
오오 저렇게 쓰면서도 다른 방법으로 표현하면 좋겠다고 생각했는데 너무 좋은 것 같아요 ㅎㅎ